lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -14,6 +15,12 @@ def initialize(serializer, options = {})
           else
             @fieldset = options[:fieldset]
           end
+
+          if links = options.delete(:links)
```

Comment:
or just `options.fetch(:links) { Hash.new }`

But, it shouldn't be adding links if none are given, right?

Also, why mutate the options hash? (note: did not look carefully)